src/session/cookie-store/index.ts

```diff
@@ -18,54 +18,65 @@ export default class CookieSessionStore implements ISessionStore {
    * @param req HTTP request
    */
   async read(req: IncomingMessage): Promise<ISession | null> {
-    const { cookieSecret, cookieName } = this.settings;
+    try {
+      const { cookieSecret, cookieName } = this.settings;
 
-    const cookies = parseCookies(req);
-    const cookie = cookies[cookieName];
-    if (!cookie || cookie.length === 0) {
-      return null;
-    }
+      const cookies = parseCookies(req);
+      const cookie = cookies[cookieName];
+      if (!cookie || cookie.length === 0) {
+        return null;
+      }
+
+      const unsealed = await Iron.unseal(cookies[cookieName], cookieSecret, Iron.defaults);
+      if (!unsealed) {
+        return null;
+      }
 
-    const unsealed = await Iron.unseal(cookies[cookieName], cookieSecret, Iron.defaults);
-    if (!unsealed) {
+      return unsealed as ISession;
+    } catch (error) {
+      console.error(error);
```

Comment:
Let's avoid logs since this is a reusable library.  It would bug me if libraries started muddying up my console.  If we feel like something like this is necessary, we should look at a debug tool that can be easily switched on/off.